course-backend/course-content/apps/api/src/app/users/mappers/users.mappers.ts

```diff
@@ -0,0 +1,27 @@
+import {RegisterUserDto} from "../dto/register-user.dto";
+import {UsersModel} from "../model/users.model";
+import {UserDto} from "../dto/user.dto";
+import bcrypt from "bcrypt";
+import {UsersRole} from "../model/users-role";
+
+
+export class UsersMappers{
+  static mapRegisterUsertoModel(dto:RegisterUserDto):UsersModel{
+    const hashed_password = bcrypt.hash(dto.password,Math.random());
+    return new UsersModel({
```

Comment:
**suggestion:** add the hashed password to the model and the dto, take care it is saved